editor/js/Sidebar.Material.js

```diff
@@ -893,6 +893,8 @@ Sidebar.Material = function ( editor ) {
 
 			}
 
+			signals.materialChanged.dispatch( material );
```

Comment:
Yes, better. Still needs to run `updateUI` to keep `null` maps from being enabled, though.